packages/openapi3/src/types.ts

```diff
@@ -695,3 +697,391 @@ export interface Ref<T> {
 }
 
 export type Refable<T> = Ref<T> | T;
+
+export type JsonSchemaType =
+  | "array"
+  | "boolean"
+  | "integer"
+  | "null"
+  | "number"
+  | "object"
+  | "string";
+
+export type JsonSchema<AdditionalVocabularies extends {} = {}> = AdditionalVocabularies & {
+  /**
+   * The JSON type for the schema.
+   * If the value is an array, then an instance validates successfully if its type
+   * matches any of the types indicated by the string in the array.
+   *
+   * @see https://json-schema.org/draft/2020-12/json-schema-validation#name-type
+   */
+  type?: JsonSchemaType | JsonSchemaType[];
+
+  /**
+   * The extending format for the previously mentioned type.
+   */
+  format?: string;
+
+  /**
+   * This provides an enumeration of all possible values that are valid
+   * for the instance property. This MUST be an array, and each item in
+   * the array represents a possible value for the instance value. If
+   * this attribute is defined, the instance value MUST be one of the
+   * values in the array in order for the schema to be valid.
+   *
+   * @see https://json-schema.org/draft/2020-12/json-schema-validation#name-enum
+   */
+  enum?: (string | number | boolean | null)[];
+
+  /**
+   * Functionally equivalent to "enum" with a single value.
+   * An instance validates successfully if its value is equal to the value of this keyword.
+   *
+   * @see https://json-schema.org/draft/2020-12/json-schema-validation#name-const
+   */
+  const?: any;
```

Comment:
should that be unknown instead of any

Reply:
Yeah that's fair, when I was copying these over I took the description _too_ literally 😆 
> The value of this keyword MAY be of __any__ type